integration-tests/bats/sql-server-schema-fragments.bats

```diff
@@ -1,86 +0,0 @@
-#!/usr/bin/env bats
-load $BATS_TEST_DIRNAME/helper/common.bash
-load $BATS_TEST_DIRNAME/helper/query-server-common.bash
```

Comment:
set head is deprecated now?

Reply:
Yeah. Tests didn't work so Aaron said delete em.